pom.xml

```diff
@@ -44,6 +34,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.11.1</version>
```

Comment:
Was there something that required the version to be called out?

Reply:
I don't remember why I added this, but it can be removed, I believe I was troubleshooting something. It's no longer relevant.